preview-site-src/elements/admonition.adoc

```diff
@@ -2,6 +2,7 @@
 :keywords: admonition, admonitions, warning, section, notes
 :page-component-name: elements
 :page-support-status: eolScheduledFeature
+:page-info-pills: anypoint-pricing-core, anypoint-pricing-titanium, undefined-pill
```

Comment:
is this here to demonstrate that if the pill is undefined, it's not added to the page?

Reply:
Yep! Exactly.